Assets/MixedRealityToolkit.SDK/Features/Input/Events/ManipulationEventData.cs

```diff
@@ -14,7 +14,7 @@ public class ManipulationEventData
         /// <summary>
         /// The object being manipulated
         /// </summary>
-        public ManipulationHandler ManipulationSource { get; set; }
+        public GameObject ManipulationSource { get; set; }
```

Comment:
breaking change?

Reply:
will this still work if you have a monobehavior link instead of gameobject @MenelvagorMilsom ? that way you might be able to avoid the breaking change?

Reply:
Maybe, will there not still have to be a cast to either `ManipulationHandler` or `ObjectManipulator`?